multi_imbalance/utils/data.py

```diff
@@ -82,25 +86,31 @@ def load_arff_dataset(path: str, one_hot_encode: bool = True, return_non_cat_len
         return X.to_numpy(), y
 
 
-def load_datasets_arff(return_non_cat_length=False, dataset_paths=None):
+def load_datasets_arff(
+    return_non_cat_length: bool = False, dataset_paths: Union[str, None] = None
+) -> OrderedDict:
     if dataset_paths is None:
-        dataset_paths = glob.glob(f'{get_project_root()}/data/arff/*')
+        dataset_paths = glob.glob(os.path.join(get_project_root(), "data", "arff", "*"))
 
     datasets = OrderedDict()
     for path in sorted(dataset_paths):
-        dataset_file = path.split('/')[-1]
-        dataset_name = dataset_file.split('.')[0]
+        dataset_file = os.path.basename(path)
+        dataset_name = os.path.splitext(dataset_file)[0]
```

Comment:
I am actually more fan of Path from pathlib: 
`dataset_name = Path(path).name`
just like that, but the way it is now is perfectly fine too

Reply:
Done